guides/user/canary/best-practices/index.md

```diff
@@ -16,6 +16,111 @@ critical metric fails, but the rest pass, the group gets a passing score overall
 You can put a critical metric in a group of only one to ensure that if it fails,
 the whole group fails every time.
 
+## Compare canary against baseline, not against production
+
+You might be tempted to compare the canary deployment against your current
+production deployment. Instead always compare the canary against an equivalent
+baseline, deployed at the same time.
+
+The baseline uses the same version and configuration that is currently running
+in production, but is otherwise identical to the canary:
+
+* Same time of deployment
+* Same size of deployment
+* Both receive the same amount of traffic
+
+In this way, you control for version and configuration only, and you reduce
+factors that could affect the analysis, like the cache warmup time, the heap
+size, and so on.
+
+## Run the canary for enough time
+
+You need at least 50 pieces of time series data per metric for the statistical
+analysis to produce accurate results. That is 50 data points per canary run,
+with potentially several runs per canary analysis. In the end, you should plan
+for canary analyses several hours long.
+
+You will need to tune the time parameters to your particular application. A good
+starting point is to have a canary lifetime of 3 hours, an interval of 1 hour
+and no warm-up period (unless you already know your application needs one).
+This gives you 3 canary runs, each 1 hour long.
+
+## Carefully choose your thresholds
+
+You need to configure two thresholds for a canary analysis:
+
+* marginal
+
+  If a canary run has a score below than this threshold, then the whole canary
+  fails.
+
+* pass
+
+  The last canary run of the analysis must score higher than this threshold for
+  the whole analysis to be considered successful. Otherwise it fails.
+
+These thresholds are very important for the analysis to give an accurate result.
+You need to experiment with them, in the context of your own application, its
+traffic and its metrics.
+
+Good starting points:
+
+* marginal threshold of 75
+* pass threshold of 95
+
+## Carefully choose the metrics to analyze
+
+You can get started with a single metric, but in the long run, your canaries
+will use several.
+
+Use a variety of metrics that reflect different aspects of the health of your
+application. Use these three out of the four "golden signals," as defined in
+the [SRE Book](https://landing.google.com/sre/book/chapters/monitoring-distributed-systems.html):
+
+* latency
+* errors
+* saturation
+
+If you consider some other specific metrics critical, place them in their own
```

Comment:
There is a 'criticality' setting that can be applied to individual metrics for this purpose.